script.module.websocket/addon.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.module.websocket" name="websocket library" version="0.57.0+matrix.1" provider-name="pkscout">
+  <requires>
+    <import addon="xbmc.python" version="3.0.0" />
+    <import addon="script.module.six" version="1.14.0+matrix.2" />
+  </requires>
+  <extension point="xbmc.python.module" library="lib" />
+  <extension point="xbmc.addon.metadata">
+    <assets>
+      <icon>icon.png</icon>
+    </assets>
+    <summary lang="en_GB">websocket library repacked for Kodi</summary>
+    <source>https://github.com/websocket-client/websocket-client</source>
+    <website>https://github.com/pkscout/script.module.websocket</website>
+    <platform>all</platform>
+    <license>LGPL</license>
```

Comment:
Please use the correct spdx identifier https://spdx.org/licenses/

Reply:
fixed in 5ee4ea2

I discovered while checking this that I had an old readme somehow. The license changed a couple years ago, so I got the correct readme and license and updated the addon.xml.  I also verified that all the python files were correct just to make sure.